frontend/packages/design-system/src/themes/Button.ts

```diff
@@ -80,4 +86,10 @@ export default {
       };
     },
   },
+  sizes: {
+    nrc_10: {
+      height: 'nrc_10',
```

Comment:
Why do we have the `nrc` prefix here? Does this mean when we change `size` we need to do `size="nrc_10"`? If so that doesn't feel right.

Reply:
yeah that would mean `size="nrc_10"`
I thought it would be better to not overwrite the space and size values of native base and better to add our custom ones, so I needed a new name, and I went with `nrc_` because I thought this design system is supposed to be used nrc-wide.
do you disagree with the custom values, or with the name? or both? :sweat_smile:

Reply:
I think it's better to override the defaults. Say you're using native base with no theme and you have:
```
<Component size="10" />
```
Then you decide you want to use the NRC theme. After applying the theme your component won't be using the nrc theme without going through the codebase and changing it to `nrc_10`.

I also just think it doesn't read as well and could lead to mistakes where people use `size="10"` as it is more intuitive, but is the incorrect style.

Reply:
@neb42 I would agree. but I felt that our spacing/sizing concept was not mature enough to replace it.
native base has a list of 43 different values, a system based on 4. I could replace everything by our system of 5, but implementing the components, I see that we do also have margins and paddings of 2px or 12px and other numbers not divisible by 5. so I'd have to add new values to the theme, but what do I name those? if "1"=5px, "2"=10px, what is 2 pixels? they do work with fractions, but I'm not very happy naming a variable "5/2.5" to arrive at 2px.

if you have an idea on how to integrate these non-5 values into the system name-wise, I'm happy to do that!

Reply:
Yeah I guess this is why I don't like using numbers for token values. I think for now leaving the spacing tokens as they are is ok. But I would change the button size. I don't think it should be using numbers. We should stick with `"xs", "sm", "md", "lg"` or a reduced set of that for what we need.

Reply:
I think we want to make using our design system as similar to using native base as possible. So if someone reads the native base docs they could more or less use the nrc theme.